src/remove-task.js

```diff
@@ -9,4 +11,22 @@ export default class Delete {
   }
 
   // Delete all completed tasks
+  static deleteAll(arr) {
+    const completedTasks = [];
+    arr.forEach((element) => {
+      if (element.completed === 'true') {
+        completedTasks.push(element);
+      }
+    });
+    let counter = 0;
+    completedTasks.forEach((element) => {
+      arr.splice(element - counter, 1);
+      counter += 1;
+    });
+    for (let i = 0; i < arr.length; i += 1) {
+      arr[i].index = i;
+    }
+    setStorage(arr);
+    return arr;
+  }
```

Comment:
Kindly fix this small problem, If there 3 tasks for example and I marked task 2 completed 
![image](https://user-images.githubusercontent.com/24703171/159946228-b44828af-0ff1-4508-8625-f5a9cacc0d4e.png)
then press `clear all completed` the first task will be removed instead of the second one.
![image](https://user-images.githubusercontent.com/24703171/159946640-6bf0ac35-3f5b-4eb0-9f54-74d91b0ddb06.png)